### PR TITLE
Update Course Controller/Router

### PR DESCRIPTION
**Pull request by ianwong1999**
LGTM

### CODE REVIEW

**Comment by ianwong1999** [comment]
> Did you just comment LGTM to yourself

yea why not. It looks good to me.

File: server/src/course/model.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
+
 import mongoose from 'mongoose';
-import bcrypt from 'bcrypt-nodejs';
 
 // Define the model
 const Schema = new mongoose.Schema({
-    couseCode: String,
+    courseCode: String,
     semesterId: String,
     time: {
```
Comment (jeremy624) [inline server/src/course/model.js]: I think time should be an array of (row, column)?

Comment (ianwong1999) [reply server/src/course/model.js]: bro im surprised you actually did read it.

**Comment by percywtc** [comment]
rebased, i think good to merge? @ianwong1999

**Comment by ianwong1999** [comment]
> rebased, i think good to merge? @ianwong1999

k

**Pull request merged**